src/actions/acquire_stepped_freq_tdomain_iq.py

```diff
@@ -201,24 +260,20 @@ def build_sigmf_md(self, task_id, measurement_params, data):
         time_domain_avg_power += (
             10 * np.log10(1 / (2 * 50)) + 30
         )  # Convert log(V^2) to dBm
-        sensor_overload = (
-            time_domain_avg_power
-            > self.sdr.radio.sensor_calibration_data["1db_compression_sensor"]
-        )
+        sensor_overload = False
+        if self.sdr.radio.sensor_calibration_data["1db_compression_sensor"]:
```

Comment:
Same comment as for M4S action - may evaluate to False with a valid compression level if it's exactly 0.0dBm.

Reply:
Changed to check `is not None`